storybook/stories/_docs/foundation_themes.stories.mdx

```diff
@@ -2,9 +2,62 @@ import { Meta, Story, Canvas } from '@storybook/addon-docs'
+## Internal Theme
+
+Apply a body class of `.ThemeInternal` to your build to reflect JPL's internal color scheme. The difference between the internal theme and the default theme is subtle: for the most part, only link colors are changed to `jpl-sky-blue` and `jpl-sky-blue-dark`.
+
+```html
+<body class="ThemeInternal">
+  <!-- page layout and components go here -->
+</body>
+```
+
+The following story shows a direct comparison of how BaseLink changes when using `.ThemeInternal` vs the default theme (no body class).
+
+<Canvas>
+  <Story name="Internal">
+    {`
+      <div>
+        <div class="ThemeInternal">
+          <!-- BaseLink -->
+          <div>
+            <a
+              href="#"
+              target=""
+              rel=""
+              class="BaseLink group cursor-pointer -default underline text-theme-red can-hover:hover:text-theme-red-hover"
+            >
+              Internal Theme: BaseLink 
+            </a>
+          </div>
+          <!-- /BaseLink -->
+        </div>
+        <div>
+          <!-- BaseLink -->
+          <div>
+            <a
+              href="#"
+              target=""
+              rel=""
+              class="BaseLink group cursor-pointer -default underline text-theme-red can-hover:hover:text-theme-red-hover"
+            >
+              Default Theme: BaseLink
+            </a>
+          </div>
+          <!-- /BaseLink -->
+        </div>
+      </div>
+    `}
+  </Story>
+</Canvas>
+
+By default, all stories displayed within this Storybook do <em>not</em> use the internal theme. However, you can set the theme by clicking on the "picture" icon in the toolbar menu. You can test this with the above story--changing the theme to use ThemeInternal should make both links appear blue. Note: some documentation pages use iframes to display components, which circumvents the theme switcher. Thus, for the most accurate representation of the internal theme, we recommend checking the component in the Canvas tab.
+
+## Light & Dark Themes
+
+The light and dark themes in Explorer 1 are structured differently from the "Internal Theme," and are **not intended for use as a body class**. These themes are used to set the context per component or template, and also our [adaptive text colors](#adaptive-text-colors). Explorer 1 defaults to a light color scheme, but you may wish to provide site editors or visitors with the option to set a dark theme on a particular page or throughout your site. To handle switching between light and dark themes, a parent class will need to be set. Individual components that have theme-specific styling will then use these classes as ancestor selectors when setting colors.
```

Comment:
> you may wish to provide site editors or visitors with the option to set a dark theme on a particular page or throughout your site.

This line carried over from the previous intro to this page seems to conflict with the guidance that these are not intended for use as a body class, but I may be misunderstanding.

I suppose the root question I have is: Why not use these as a class on `body`?

Reply:
Answering my own question – I guess it's due to what you emphasized below, that most components don't have a dark variant. I think I would just rework this paragraph a bit to indicate that they should probably only ever be used on the wrapper for an individual component, if I've got it right that that is our current position.